modules/reference/pages/console/record-deserialization.adoc

```diff
@@ -1,19 +1,18 @@
-= Deserialization in Redpanda Console
-:description: Learn how Redpanda Console deserializes messages.
+= Deserialization
```

Comment:
I would undo these changes. This page is about how Redpanda Console deserializes messages.

If we want to include these details in the cloud docs, we should conditionalize the product name and exclude all the self-hosted configuration sections.

Reply:
@JakeSCahill and @micheleRP, I've replaced the product name with the {ui} attribute and added a conditional paragraph in the [Schema Registry section](https://deploy-preview-586--redpanda-docs-preview.netlify.app/current/reference/console/record-deserialization/#schema-registry).